testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -531,9 +522,7 @@
 When(/^I install the package download endpoint pillar file on the server$/) do
   filepath = '/srv/pillar/pkg_endpoint.sls'
   uri = URI.parse($custom_download_endpoint)
-  content = "pkg_download_point_protocol: #{uri.scheme}\n"\
-            "pkg_download_point_host: #{uri.host}\n"\
-            "pkg_download_point_port: #{uri.port}"
+  content = "pkg_download_point_protocol: #{uri.scheme}\npkg_download_point_host: #{uri.host}\npkg_download_point_port: #{uri.port}"
```

Comment:
This is also much harder to read FMPOV.

Reply:
Well, in this particular case, as it's not that long line, I like it as it's automatically refactored.
But I will revert, as I already reverted others similar and the Cop is disabled.